### DISCUSSION TITLE
Requesting product Logo icon to paste

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1204] [Request: Icon] [answered]
IoT BU likes following product logo icons to be added to paste.

**List of Logo Icons:**
programmable-wireless-logo-red
supersim-logo-red
narrowband-logo-red
logo-embedded-sim-red
trust-onboard-logo-red

**Location** to find the product marketing logo:
https://library.twilio.com/share/05F3D651-DE1B-456D-B1B60148845CCB64/

**Description** why we need product logos. IoT has 3 products under its umbrella. We will be using product logos in the SIMs list table to identify the form factor of the SIMs.

**Comment by serifluous** [comment]
Hey @ishah-twilio ! We'll add these to Figma to start, and then to Sketch as part of our March 9 release. Because these are all product logos specific to IoT, we'll give the icons these names:

- ProductInternetOfThingsWirelessIcon
- ProductInternetOfThingsSuperSIMIcon
- ProductInternetOfThingsNarrowbandIcon
- ProductInternetOfThingsEmbeddedSIMIcon
- ProductInternetOfThingsTrustOnboardIcon

We'll follow up with you to review them, and then reply here again when it's ready for engineering use.

**Reply by ishah-twilio** [reply]
Thank you so much @serifluous.

Does that mean svg is accessible by engineers for their need once it will be added? Or there is another process that we need to follow? Not sure how my eng team can have access to these icons as well. Greatly appreciate your reply back.

**Reply by serifluous** [reply]
Yep, it may take up to 3 weeks to get them added so that it's accessible for your engineers—we'll need to add it into our sprint. If you need it faster than that, I can send you the formatted SVGs, and your engineers can add it to the Paste repo themselves by following these [self-service instructions](https://paste.twilio.design/icons/how-to-add-an-icon/#adding-the-icon-to-the-paste-repository).

**Reply by victoriahu** [reply]
Hi @serifluous can you slack me the formatted SVGs? I'm going to try to add it to the Paste repo myself

**Reply by ishah-twilio** [reply]
@victoriahu -- I will give you SVG.

**Reply by serifluous** [reply]
@victoriahu @ishah-twilio I'll send them to you both. There's probably some adjustment that needs to be made on the original SVGs.

**Reply by ishah-twilio** [reply]
@serifluous - thank you.

**Comment by serifluous** [comment, marked as answer]
Rounding this Discussion out: @ishah-twilio reviewed the icons and we got the SVGs to @victoriahu to submit a PR.
